reemgawad/brunch-time
# Places API

**What changed:**
- Using different Places API endpoints to fetch brunch restaurant data in Montreal
- Creating Restaurant objects using fetched data

**How to test:**
Go to /restaurants
- Should see over ~30 restaurants.
- Should be brunch restaurants.
- Each should have an image displayed.

Go to a specific restaurant
- Details should be displayed correctly.
